.travis.yml

```diff
@@ -90,6 +90,12 @@ jobs:
       env:
         - TEST='raiden/tests/unit'
 
+    - stage: test
+      << *job-template-test
+      env:
+        - TEST='raiden/tests/fuzz'
+        - TRANSPORT_OPTIONS='--hypothesis-show-statistics'
```

Comment:
Hm not the cleanest way to 'abuse' the TRANSPORT_OPTIONS here but I guess since travis is on the way out in any case it doesn't matter that much.

Reply:
That's what I thought, otherwise I would have renamed them to `ADDITIONAL_OPTIONS` or something.